rfc6962-bis.xml

```diff
@@ -766,7 +776,7 @@ messages.
           <t hangText="error_code:">
             An error code readable by the client. Some codes are generic and are detailed here. Others are detailed in the individual requests. Error codes are fixed text strings.
             <list style="hanging">
-              <t hangText="not compliant">
+              <t hangText="'not compliant'">
```

Comment:
Is the log meant to include the single quotes when it outputs this error?  I'm assuming not, in which case I think adding the single quotes in the document creates ambiguity.